lib/json/logger.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Json.Logger do
+  @moduledoc """
+  Exposes separate log level configuration so developers can set logging
+  verbosity for json library
+
+  To configure log level only for json library, add folowing line in config file
+
+      use Mix.Config
+      # to make json be very verbose
+      config :json, log_level: :debug
+      # to make json be silent
+      config :json, log_level: :error
+  """
+  require Logger
+  @levels [:debug, :info, :warn, :error]
+  @level Application.get_env(:json, :log_level, :info)
+
+  @allowed_levels @levels
+                  |> Enum.reverse()
+                  |> Enum.reduce_while([], fn l, acc ->
+                    if l == @level do
+                      {:halt, [@level | acc]}
+                    else
+                      {:cont, [l | acc]}
+                    end
+                  end)
+                  |> Enum.reverse()
+
+  @doc false
```

Comment:
No need to remove the doc generation for this since its a non-private function.

Reply:
this is done too.

Reply:
👍